src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Lottery {
+    private Ball ball;
+    private ColorSupplier color;
+    private NumberBall numberBall;
+
+    public Lottery(Ball ball, ColorSupplier color, NumberBall numberBall) {
+        this.ball = ball;
+        this.color = color;
+        this.numberBall = numberBall;
+    }
+
+    public Ball getRandomBall() {
+        ball.setColor(color.getRandomColor());
+        ball.setNumber(numberBall.getRandomNumber());
+        return ball;
```

Comment:
try to create ball using constructor instead of setters

Reply:
https://prnt.sc/ext-4wc0iGU0
created